Rakefile

```diff
@@ -139,8 +139,11 @@ namespace :cordova do
   desc "Cordova build {platform}"
   task build: :prepare do
     Dir.chdir(CORDOVA_PATH) do
-      build = (environment == "staging" && platform == 'android') ? "debug" : "release"
-      system({"ENVIRONMENT" => environment}, "cordova compile #{platform} --#{build} --device")
+      team_id = ENV['IOS_DEVELOPMENT_TEAM_ID']
+      provisioning_profile = ENV['PROVISIONING_PROFILE_STAGING']
+      provisioning_profile = ENV['PROVISIONING_PROFILE_PROD'] if(environment === 'production')
```

Comment:
@shivanibhanwal  You'll need to add these variables to CI env.
For that you'll need to create provisioning profile for staging and prod using development certificate and replace current pro. profile with these new ones. And use hash of that provisioning profile to CI env variables.

Reply:
I have already added these ENV variables in CI for the staging. Then only build has passed on circleCI. Also I have uploaded the certificate too. But I did not add it for Prod as I am still concerned will iOS support Developer license for PROD.